crowdin.yml

```diff
@@ -0,0 +1,100 @@
+#
+# Your Crowdin credentials
+#
+"project_id_env" : "CROWDIN_PROJECT_ID"
+"api_token_env" : "CROWDIN_PERSONAL_TOKEN"
+"base_path" : "."
+"base_url" : "https://api.crowdin.com"
+
+#
+# Choose file structure in Crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
+#
+# Files configuration
+#
+files: [
+  {
+     "source" : "/services/src/main/resources/locale/addon/Gamification_en.properties",
+
+    "translation" : "%original_path%/%file_name%!_%locale_with_underscore%.%file_extension%",
+    "translation_replace" : {
+      "_en!":"",
+      "de_DE": "de",
+      "en_US": "en",
+      "es_ES": "es_ES",
+      "fr_FR": "fr",
+      "it_IT": "it",
+      "pt_BR": "pt_BR",
```

Comment:
In the translation.zip file, we have much more languages than theses one.
I think we must put all in this list to make it work.
```suggestion
      "ar_SA":"ar",
      "ar_OM":"aro",
      "az_AZ":"az",
      ....
      "de_DE": "de",
      "en_US": "en",
      "es_ES": "es_ES",
      "fr_FR": "fr",
      "it_IT": "it",
      "pt_BR": "pt_BR",
```
I put just the main ones. 
As this is only for download action, it is possible to test this specific action : 
launch it, but not merge. So we can confirm that in the created merge branch, all languages are ok, and fix if necessary
Should we merge before test ? or can we test before merge ?

Reply:
the merge should be done in order to apply the github actions. I may comment the merge part in the begining